addons/l10n_cl_website_sale/static/src/js/fields/l10n_cl_fields.js

```diff
@@ -0,0 +1,70 @@
+/** @odoo-module **/
+import { WebsiteSale } from 'website_sale.website_sale';
+
+WebsiteSale.include({
+    events: Object.assign(WebsiteSale.prototype.events, {
+        'change input[name="l10n_cl_sii_taxpayer_type"]': '_onChangeClickAddTypeDocumentCl',
+    }),
+    /**
+     * @override
+     */
+    start: function () {
+        var def = this._super.apply(this, arguments);
+        if ($('input[id="radio_ticket"]').length > 0 && this.target.baseURI.indexOf('/shop/address')){
```

Comment:
New code should probably use as less jquery as possible.

Reply:
@hea-bmya could you please check this comment?

@Danisan

Reply:
Hi, I optimize some jquery consult, I wait for your comments.

@Williambraecky @Danisan @fvz-odoo

Reply:
@hea-bmya this is still using jquery..
Moreover having `input[name=vat]` as a querySelector is kinda dangerous, `id=` something is ok since ids are meant to be unique.
Please change the code to use vanilla javascript [this might help you](https://tobiasahlin.com/blog/move-from-jquery-to-vanilla-javascript/)

Reply:
Hi @Williambraecky grateful for the link, the changes are done!

@Danisan @fvz-odoo @jco-odoo